src/stories/Forms/Toggle/Toggle.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { InputWrapper } from '../../../index';
+import { Toggle } from '.';
+
+export default { component: Toggle };
+
+const Template = ({ alert, children, hint, id, isRequired }) => {
+  return (
+    <InputWrapper alert={alert} hint={hint} id={id} isRequired={isRequired}>
+      <Toggle>{children}</Toggle>
```

Comment:
Children added in story but no support in Toggle component. I have another comment about this further in the review that explains more.